Doc/library/math.rst

```diff
@@ -95,7 +104,7 @@ noted otherwise, all return values are floats.
 :func:`sinh(x) <sinh>`                                Hyperbolic sine of *x*
 :func:`tanh(x) <tanh>`                                Hyperbolic tangent of *x*
 
-**Special functions**
+**Error and gamma functions**
```

Comment:
I think it's fine to keep "Special functions" because in mathematics we call them "Special functions": https://en.wikipedia.org/wiki/Special_functions.

Reply:
I didn't know that, thank you for the information.

Reply:
No worries! I know that the C++ reference uses Error & Gamma functions but I think Python decided to go for the more general category of functions.

Reply:
I don't think we will add anything beyond C standard.

Reply:
"Error and gamma functions" is the title used in the C documentation page. I initially changed it because I found it more explicit, but I didn't know the mathematical meaning of "Special functions". Both seem fine, so maybe status quo wins?